src/mudlet.cpp

```diff
@@ -2058,9 +2053,10 @@ void mudlet::slot_show_connection_dialog()
     auto pDlg = new dlgConnectionProfiles(this);
     connect(pDlg, SIGNAL(signal_establish_connection(QString, int)), this, SLOT(slot_connection_dlg_finished(QString, int)));
     pDlg->fillout_form();
-    if (pDlg->exec() == QDialog::Accepted) {
```

Comment:
It did not block the rest of the application - `exec()` has its own event loop. It was just modular so it blocked interaction with the rest of the application. This can easily be tested by getting a copy of the latest release.

Yes, everything still works fine, I test my PRs before I submit them and I've said in the previous comment that I've tested it as well... is this good to be merged?